readme.md

```diff
@@ -353,6 +353,17 @@ Type: `Object<string, Function[]>`
 
 Hooks allow modifications during the request lifecycle. Hook functions may be async and are run serially.
 
+###### hooks.init
+
+Type: `Function[]`<br>
+Default: `[]`
+
+Called with plain [request options](#options), right before their normalization. This is especially useful in conjunction with [`got.extend()`](#instances) and [`got.create()`](advanced-creation.md) when the input needs custom handling.
+
+See the [Request migration guide](migration-guides.md#breaking-changes) for an example.
+
+**Note**: this hook must be synchronous!
```

Comment:
It must be synchronous - otherwise `normalizeArguments` would be asynchronous, so errors thrown [here](https://github.com/sindresorhus/got/blob/master/source/create.js#L32) wouldn't be caught.